README.md

```diff
@@ -25,8 +25,23 @@ Usage
 ember addon my-addon -b @embroider/addon-blueprint --yarn
 ```
 
+If you want release-it behavior, (specifically provided by `create-rwjblue-release-it-setup`),
+use the `--release-it` flag
+
+```bash
+ember addon my-addon -b @embroider/addon-blueprint --yarn --release-it
+```
+
+NOTE: this addon is not published, so instead of specifying `@embroider/addon-blueprint`,
+you'll want to clone this repo, and pass the path to this repo to `-b`.
+
+```bash
+ember addon my-addon -b ../path/to/addon-blueprint
+```
```

Comment:
The section above can be removed I think! First, it has been published now. But I also changed the invocation example recently to use a github reference (and not npm), see #20. That way you also don't need to clone, which is an annoying extra step. Also unrelated to the actual changes in this PR :wink:

Reply:
oh nice! I didn't know that was supported! legit